docs/tasks/semester-start-end-tasks.qmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "DataHub Semester Start and End Tasks"
+---
+
+This document outlines the tasks for preparing DataHub for the start of a semester and for concluding semester activities.
+
+## Semester Start Tasks
+
+### 1. Setup and Configuration
+- [ ] **Update JupyterHub version**: Check for and apply updates to [Z2JH](https://z2jh.jupyter.org/en/latest/changelog.html) JupyterHub (if required)
+- [ ] **Update JupyterLab/Notebook version**: Check for and apply updates to JupyterLab and Notebook (if required)
+- [ ] **Bump Replica Values**: Bump replica values in values.yaml
+- [ ] **Set Node Count**: Set the appropriate number of node count for each node pool in GKE console
+- [ ] **Otter Grader Version Upgrade**: Check for and apply updates to Otter Grader across all hubs (if required)
+
+### 2. User Management
+- [ ] **Identify Unused Hubs**: Identify hubs that will not be used during a particular semester
+- [ ] **Send Onboarding Email**: Send onboarding instructions to all the instructors and GSIs added to `datahub-announce` listserv
+
+---
+
+## Semester End Tasks
+
+### 1. Operational Tasks
+- [ ] **Update Ubuntu**: Check for and apply updates to Ubuntu (if required)
+- [ ] **Update Python**: Check for and apply updates to Python (if required)
+- [ ] **Update Kubernetes**: Check for and apply updates to Kubernetes (if required)
+- [ ] **Reduce Resources**: 
+  - Scale down node placeholder pods to 0
+  - Reduce the number of nodes allocated for each node pool 
+- [ ] **Clear SQLite Db**: Clear SQLite database that caches user info for hub pods
+- [ ] **Delete Users**: Delete users from ORM database
```

Comment:
this step and the one before it are actually the same

Reply:
Oops, fixing it